packages/components/src/Datalist/Datalist.component.test.js

```diff
@@ -163,7 +163,6 @@ describe('Datalist component', () => {
 		// then
 		const payload = { value: 'foobar' };
 		expect(onChange).toBeCalledWith(expect.anything(), payload);
-		expect(wrapper.find(Typeahead).props().items).toBe(null);
```

Comment:
Can you update the test description accordingly please ? (no reset suggestion anymore)
Something like `should trigger change callback on blur ....`